renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "github>mightyiam/renovate-config:js-lib"
```

Comment:
https://docs.renovatebot.com/config-presets/
https://github.com/mightyiam/renovate-config
@feross I don't have permission to install [the GitHub app](https://github.com/marketplace/renovate).